src/component/text/Word.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from "react";
+import { Letter } from "./Letter";
+import "./css/Word.css";
+
+interface Props {
+  word: String;
+  input: string;
+  onCurrentWord: boolean;
+  currentWordSubstring: string;
+  onPastWords: boolean;
+}
+
+export const Word = (props: Props) => {
+  const generateLetters = () =>
+    //for (let letterIndex = 0; letterIndex < props.word.length; letterIndex++)
```

Comment:
If this isn't used we should remove it. Generally we don't want commented out code in our project. We can always add it back if we need it.

Reply:
@swcraig fixed with a rebase, thank you